### PR TITLE
Update tools.yml

### PR DESCRIPTION
**Pull request by htadashi**
Added openapi-fuzzer

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -1538,6 +1538,15 @@
   link: https://42crunch.com/
   v2: true
   v3: true
+  
+- name: openapi-fuzzer
+  category: security
+  language: Rust
+  description:
+    Black-box fuzzer that fuzzes APIs based on OpenAPI specification. All you need to do is to supply URL of the API and its specification.
```
Comment (inverse) [inline _data/tools.yml:1546]:
```suggestion
    Black-box fuzzer that fuzzes APIs based on OpenAPI specification.
```

**Review by inverse — APPROVED** [review]
Description could be simplfied - aside that LGTM

File: _data/tools.yml [hunk 2]
```diff
@@ -1538,6 +1538,15 @@
   link: https://42crunch.com/
   v2: true
   v3: true
+  
+- name: openapi-fuzzer
+  category: security
+  language: Rust
+  description:
+    Black-box fuzzer that fuzzes APIs based on OpenAPI specification.
```
Comment (philsturgeon) [inline _data/tools.yml:1546]: Can we try and explain what this fuzzer does without saying its a fuzzer that fuzzes? I know what fuzzing means in some contexts, but maybe not this one, and some people have never heard of fuzzing at all. Why do they need this in their life.

Comment (htadashi) [reply _data/tools.yml:1546]: @philsturgeon Thanks for the remark. How about this description?
"Based on OpenAPI specification, openapi-fuzzer provides random data as inputs to the API endpoins in order to find bugs"

Comment (philsturgeon) [reply _data/tools.yml:1546]: Brilliant yeah!

Comment (htadashi) [reply _data/tools.yml]: Done :)

**Pull request merged**